test/crio-wipe.bats

```diff
@@ -32,35 +32,27 @@ function run_crio_wipe() {
 # test_crio_wiped_containers checks if a running crio instance
 # has no containers or pods
 function test_crio_wiped_containers() {
-	run crictl pods -v
-	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
-
-	run crictl ps -v
-	[ "$status" -eq 0 ]
-	[ "$output" == "" ]
+	output=$(crictl pods -v)
+	[ "$output" = "" ]
+	output=$(crictl ps -v)
+	[ "$output" = "" ]
 }
 
 function test_crio_did_not_wipe_containers() {
-	run crictl pods -v
-	[ "$status" -eq 0 ]
-	[ "$output" != "" ]
+	output=$(crictl pods -v)
+	[ "$output" =! "" ]
```

Comment:
shouldn't this be
`[ "$output" != "" ]`

Reply:
😳